package.json

```diff
@@ -7,16 +7,16 @@
   "author": "Michał Pierzchała <thymikee@gmail.com>",
   "license": "BSD-3",
   "dependencies": {
-    "jest-zone-patch": "^0.0.7",
-    "jsdom": "^11.3.0",
-    "ts-jest": "^21.0.0"
+    "jest-zone-patch": "^0.0.8",
+    "jsdom": "11.5.1",
+    "ts-jest": "21.2.4"
```

Comment:
btw, why are these deps locked?

Reply:
Can we make `jsdom@^11.5.1` and `ts-jest@^21.2.4`?

Reply:
yes we can. Nothing breaks about this change. I did commit yarn.lock file for this change too